rootfs/scheduler/__init__.py

```diff
@@ -316,15 +317,41 @@ def deploy(self, namespace, name, image, command, **kwargs):
         logger.debug('deploy {}, img {}, params {}, cmd "{}"'.format(name, image, kwargs, command))
         app_type = kwargs.get('app_type')
 
+        # Fetch service
```

Comment:
This updates the app type and ports as needed, mostly only happens on first deploy tho